### PR TITLE
Introduce tool/src/gen.js to generate large box scenes.

### PR DESCRIPTION
**Pull request by pablo-mayrgundter**
// Generate 2^17 boxes, ~60MB
> node tool/src/gen.js 131072 > box-131072.ifc

Demos screens:

https://discord.com/channels/853953158560743424/1064994513241780264/1150205457378578493

### CODE REVIEW

File: tool/src/gen.js [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+function boxen(num = 10) {
+  const maxSpan = 100
+  for (let i = 1; i <= num; i++) {
+    const x = parseInt(Math.random() * maxSpan)
+    const y = parseInt(Math.random() * maxSpan)
+    const z = parseInt(Math.random() * maxSpan)
+    // eslint-disable-next-line no-magic-numbers
+    const id = i * 1000
+    // eslint-disable-next-line no-magic-numbers
+    p(box(id, [x, y, z], 500, 202, 312))
```
Comment (ConorStokes) [inline tool/src/gen.js:57]: More a note, should these be variables at the top of the file/overridable command line params?

File: tool/src/gen.js [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+/** @return {string} STEP 10303-21 closing */
+function stepClose() {
+  return `END-ISO-10303-21;
+`
+}
+
+
+/** @param {string} msg */
+function p(msg) {
```
Comment (ConorStokes) [inline tool/src/gen.js:150]: I found this being at the bottom of the file + non-descriptive name made this a bit hard to read on a single scan through. I was kind of guessing at usage. 

Maybe "print" or "log" and maybe lift the function to the top of the file so the semantics are clear?

**Review by ConorStokes — APPROVED** [review]
I'm approving this because I think the tool is small enough for the notes to not be too important, but if this expands, I think the comment about the "p" function being non-descriptively named/at the bottom of the file should be addressed for readability.

File: tool/src/gen.js (hunk 1 above)
Comment (pablo-mayrgundter) [reply tool/src/gen.js:57]: Will hoist for now and maybe CLI params in future PR when using input model.

File: tool/src/gen.js (hunk 2 above)
Comment (pablo-mayrgundter) [reply tool/src/gen.js:150]: Done.  Now that using expected fn name, leaving at bottom.

**Pull request merged**